test/spec/refinements.js

```diff
@@ -98,58 +88,50 @@ test('isDisjunctiveRefined', function(t) {
 
   var value = 'MyValue';
 
-  t.equal(helper.isDisjunctiveRefined(facet, value), false,
-    'isDisjunctiveRefined should return false for when no refinement for facet');
+  expect(helper.isDisjunctiveRefined(facet, value)).toBe(false);
 
   helper.addDisjunctiveRefine(facet, value);
-  t.equal(helper.isDisjunctiveRefined(facet, value), true,
-    'isDisjunctiveRefined should be true when facet is refined');
+  expect(helper.isDisjunctiveRefined(facet, value)).toBe(true);
 
   helper.removeDisjunctiveRefine(facet, value);
-  t.equal(helper.isDisjunctiveRefined(facet, value), false,
-    'isDisjunctiveRefined should return false when refinement was removed');
-  t.end();
+  expect(helper.isDisjunctiveRefined(facet, value)).toBe(false);
 });
 
-test('IsRefined should return true if the (facet, value ) is refined.', function(t) {
+test('IsRefined should return true if the (facet, value ) is refined.', function() {
   var helper = algoliasearchHelper(emptyClient, null, {
     facets: ['facet1']
   });
 
   helper.addRefine('facet1', 'boom');
 
-  t.equal(helper.isRefined('facet1', 'boom'), true, 'the facet + value is refined >> true');
-
-  t.equal(helper.isRefined('facet1', 'booohh'), false, 'value not refined but is a facet');
-  t.throws(_.bind(helper.isRefined, helper, 'notAFacet', 'maoooh'), 'should throw as it is not a facet');
-  t.throws(_.bind(helper.isRefined, helper, null, null), 'not valid values');
+  expect(helper.isRefined('facet1', 'boom')).toBe(true);
 
-  t.end();
+  expect(helper.isRefined('facet1', 'booohh')).toBe(false);
+  expect(_.bind(helper.isRefined, helper, 'notAFacet', 'maoooh')).toThrow();
```

Comment:
Do we want to track the errors?

Reply:
See https://github.com/algolia/algoliasearch-helper-js/pull/660#discussion_r278628453